newIDE/app/src/UI/DrawerTopBar.js

```diff
@@ -5,6 +5,7 @@ import Toolbar from '@material-ui/core/Toolbar';
 import IconButton from '@material-ui/core/IconButton';
 import Typography from '@material-ui/core/Typography';
 import Close from '@material-ui/icons/Close';
+import { Tooltip } from '@material-ui/core';
```

Comment:
I tend to prefer `import Tooltip from '@material-ui/core/Tooltip'` as it helps tree shaking which is not done automatically with this version. See https://v4.mui.com/guides/minimizing-bundle-size/

This being said, we're using so much of `material-ui/core` that I'm not sure it's worth it (it's definitely worth for `@material-ui/icons`), and should probably be linted if we want to enforce ti.